biztown/src/components/CheckSlip.js

```diff
@@ -1,136 +1,189 @@
 import React, { Component } from "react";
```

Comment:
very small stylistic thing, but on safari, this component has a but of a y-scroll. it is still v functional and looks fine enough, so feel free to disregard this comment

Reply:
I have that on chrome as well. Was planning to style everything after MVP was hit. I'll add this to the backlog.